cirq/google/optimizers/convert_to_xmon_gates.py

```diff
@@ -65,6 +64,19 @@ def _convert_one(self, op: 'cirq.Operation') -> 'cirq.OP_TREE':
 
         return NotImplemented
 
+    def _is_native_xmon_op(self, op: 'cirq.Operation') -> bool:
+        """Check if the gate within an operation is a native xmon gate.
+
+        Args:
+            op: Input operation.
+
+        Returns:
+            True if the operation is native to the xmon, false otherwise.
+        """
+        from cirq.google.devices import XmonDevice
```

Comment:
Is an inlined import acceptable style for this project? I feel like I was once critiqued for that, but I can't recall for sure :-)

Reply:
it is - unfortunately circular dependencies are forcing us to do this in quite a few places